### PR TITLE
feature#1

### PR DESCRIPTION
**Pull request by ghost**
- Add package Socialite
- Create model SocialAccount
- Create repository SocialAccountRepository
- Create controller SocialAccountController
 
1 social account - create 1 user
haven't excute to allow connect to other social account.

### CODE REVIEW

File: app/Repositories/SocialAccountRepository.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            ]);
+
+            $user = User::whereEmail($providerAccount->getEmail())->first();
+
+            if (!$user) {
+                $user = User::create([
+                    'email' => $providerAccount->getEmail(),
+                    'name' => $providerAccount->getName(),
+                    'avatar' => $providerAccount->getAvatar(),
+                    'role_id' => config('foo.USER_ID'),
```
Comment (dzungnguyenh) [inline app/Repositories/SocialAccountRepository.php:42]: [rank C] I think the naming of role is not good. And why is it calling as "role_id"?

File: config/foo.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (dzungnguyenh) [inline config/foo.php:1]: [rank C] you use file name as foo? ⭕️

File: config/foo.php [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+return [
+    'USER_ID' => 2,
```
Comment (dzungnguyenh) [inline config/foo.php:4]: [rank C] USER_ID = 2? What does it mean?

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
+@extends('layouts.app')
```
Comment (dzungnguyenh) [inline resources/views/auth/login.blade.php:1]: [rank C] plz set all labels in language files.

File: resources/views/home.blade.php [hunk 5]
```diff
@@ -0,0 +1,17 @@
+@extends('layouts.app')
```
Comment (dzungnguyenh) [inline resources/views/home.blade.php:1]: [rank C] plz set all labels in language files.

**Review by dzungnguyenh — CHANGES_REQUESTED** [review]
please check my comments

File: app/Http/Controllers/SocialAccountController.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Repositories\SocialAccountRepository;
+use Socialite;
+
+class SocialAccountController extends Controller
+{
+  /**
+   * [redirect to social page]
```
Comment (ghost) [inline app/Http/Controllers/SocialAccountController.php:15]: remove  `[` and `]` when you add function comment.

**Review by ghost — CHANGES_REQUESTED** [review]
pls check my comment

File: app/Repositories/SocialAccountRepository.php (hunk 1 above)
Comment (ghost) [reply app/Repositories/SocialAccountRepository.php:42]: "role_id", it is name is in database. I dont know if change it. Can it occur error?

**Pull request closed without merging**